packages/linea-ccip-gateway/hardhat.config.ts

```diff
@@ -18,6 +18,18 @@ const config: HardhatUserConfig = {
       },
     ],
   },
+  networks: {
```

Comment:
I think we don't have to change the hardhat config, they will do it in their own repos.